com.unity.render-pipelines.universal/Samples~/RenderPassSamples/Shaders/KawaseBlur.shader

```diff
@@ -0,0 +1,70 @@
+Shader "Custom/RenderFeature/KawaseBlur"
+{
+    Properties
+    {
+        _MainTex("Texture", 2D) = "white" {}
+    //   _offset ("Offset", float) = 0.5
+    }
+
+        SubShader
+    {
+        Tags { "RenderType" = "Opaque" }
+        LOD 100
+
+        Pass
+        {
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            // make fog work
+            #pragma multi_compile_fog
+
+            #include "UnityCG.cginc"
```

Comment:
Needs to be written in SRP shader library / URP. This is using old built-in shader library.

Reply:
Shader also needs to be stereo aware. @thomas-zeng

Reply:
The updated shader should now use the universal shader library and is now using HLSL instead of cginc.